test/02/t0265a.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+#
+# srecord - Manipulate EPROM load files
+# Copyright (C) 2013 Peter Miller
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or (at
+# your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
+TEST_SUBJECT="input efinix"
+. test_prelude.sh
+
+cat > test.in1 << 'fubar'
+S0220000687474703A2F2F737265636F72642E736F75726365666F7267652E6E65742F1D
```

Comment:
Suggest having the input and "ok" test inputs in something like ascii-hex format. It's not essential but I've found it useful when trying to spot issues in reading the file. Just a suggestion.